NAMESPACE

```diff
@@ -129,8 +129,12 @@ export(lift_lv)
 export(lift_vd)
```

Comment:
I think the other reason I think these are separate is because `map_c()` physically can't be faster than sequential calls to `map() + list_c()`, because you need all of the results from the `map()` to be able to compute the output size that we'd get from `list_c()`, and to me that suggests they shouldn't be merged together

Reply:
Yeah, agreed. I think the `length(x) == length(map*(x, f))` invariant is key.

`x |> map_vec(f)` feels a bit like this because it'll begin by being implemented as `x |> map(f) |> list_simplify()` but I think that's more of a current implementation detail. Especially if you do supply a `ptype` we could potentially implement `map_vec()` much more efficiently.